litmus-portal/backend/auth/pkg/manage/manager.go

```diff
@@ -88,6 +89,17 @@ func (m *Manager) CreateUser(user *models.User) (err error) {
 	return
 }
 
+//CreateGithubUser stores the user information
+func (m *Manager) CreateGithubUser(user *models.User) (err error) {
+	err = m.userStore.Set(user)
+	if err != nil {
+		fmt.Printf("\nerror: %v\n", err)
```

Comment:
Remove `fmt.Print` statements used anywhere. Instead use the `glog` to log anything

Reply:
`glog` is showing error, using `log` instead